website/react-magma-docs/src/pages/api/table.mdx

```diff
@@ -1064,20 +1062,16 @@ All of the [standard td attributes](https://developer.mozilla.org/en-US/docs/Web
 
 ## Table Pagination Props
 
-**This component uses `forwardRef`. The ref is applied to the Alert container `div` element.**
+**This component uses `forwardRef`. The ref is applied to the container.**
 
 All of the [global HTML attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes) can be provided as props and will be applied to the wrapping `div` element that gets rendered.
 
-### Common Props
-
-<BaseTablePaginationProps />
-
```

Comment:
`BaseTablePaginationProps` were deleted in the mdx file, maybe that's why they aren't showing up?